.github/workflows/stm32build.yml

```diff
@@ -2,13 +2,13 @@ name: C/C++ stm32 build for the open-source embedded software
 
 on:
   push:
-    branches: [ 'main' ]
-    paths: ['STM32/**', '.github/workflows/stm32build.yml']
-  pull_request: 
-    branches: [ 'main' ]
-    paths: ['STM32/**', '.github/workflows/stm32build.yml']
+    branches: ["main"]
```

Comment:
What's the reasoning with changing the one to two quotes? :)

Reply:
Shaggy defense
![image](https://github.com/copenhagenatomics/CA_Embedded/assets/141138893/805fd41e-0021-420d-8361-46d810bcfee5)

Reply:
In that case let's go back to single quotes so it is the same across all of our repos :=)